app/models/criteria/conditions/tags.rb

```diff
@@ -0,0 +1,21 @@
+module Criteria
+  module Conditions
+    Tags = Struct.new(:tag_words) do
+      extend Creatable
+
+      def prepare(relation)
+        relation.use_joins
+      end
+
+      def satisfy(relation)
+        tags = TaskParameters.tags_from(tag_words)
+        relation
+          .except(:select)
+          .select('tasks.*')
```

Comment:
Style/MultilineOperationIndentation: Align the operands of an expression in an assignment spanning multiple lines.